File: powtorzenie/warunek.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  warunek.py
# 
 
def najwieksza(a, b, c):
    
    maks = a 
    
    if b > maks:
        maks = b
        
    if c > maks:
        maks = c
        
    return maks

def main(args):
    
    a = int(input("Podaj 1. liczbę: "))
    b = int(input("Podaj 2. liczbę: "))
    c = int(input("Podaj 3. liczbę: "))
    
    maks = najwieksza(a, b, c)
    
    print("Najwiekszą liczbą jest ", maks)
    
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))

File: powtorzenie/warunek.cpp
/*
 * warunek.cpp
 * 
 */


#include <iostream>

using namespace std;

int main(int argc, char **argv)
{
    int a, b, c;
    a = b = c = 0;
    
    cout << "Podaj 1. liczbę  ";
    cin >> a;
    cout << "Podaj 2. liczbę  ";
    cin >> b;
    cout << "Podaj 3. liczbę  ";
    cin >> c;
    
    if (a>b and a>c) cout << "Największą liczbą jest " << a << endl;
    if (b>a and b>c) cout << "Największą liczbą jest " << b << endl;
    else cout << "Największą liczbą jest " << c << endl;
	
	return 0;
}

